src/renderer/Components/RecevingPackage/index.jsx

```diff
@@ -1,14 +1,18 @@
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
+import axios from "axios";
 
 import Modal from "../Modal";
 import NumberInput from "./NumberInput";
 
 function ReceivingPackage() {
   const [isModalOpen, setIsModalOpen] = useState();
+  const [numbers, setNumbers] = useState(["", "", "", "", "", ""]);
```

Comment:
```suggestion
  const SERIAL_NUMBER_LENGTH = 6;
  const [inputNumbers, setInputNumbers] = useState(new Array(SERIAL_NUMBER_LENGTH).fill(""));
```
저는 길어지더라도 명확하게 쓰는 걸 좋아하는 편인데요. 전반적으로 단순한 유효성 검사와 로직이 주를 이루는 내용이지만 그래도 별도의 utils 로 빼지 않는 맥락에서 이해하기 좋은 단어를 쓰는것이 이해하는데에 도움이 되는 것 같습니다 :) 

상수화 또한 같은 맥락에서 진행해 보았습니다. 
"" 로 6자리 라는 것이 한눈에 보이지 않았고, 6 이라는 매직넘버를 처리해 보았습니다. 
혹시 이 제안이 받아들여진다면 constants/config.js 파일 이 별도로 있으니 활용할 수도 있을 것 같습니다 :)

Reply:
비교적 6이라는 숫자가 의미하는 바가 명확하다고 생각하여 그대로 썼었지만 상수처리를 해보니 더욱 알기 쉬워지는듯합니다. 좋은 제안 감사드립니다!